app/models/ballot_item.rb

```diff
@@ -3,4 +3,17 @@ class BallotItem < ActiveRecord::Base
 belongs_to :award
 belongs_to :play
 
+  def create
+    nomination = Nomination.ranked_by_weight
+    @ballot_item = BallotItem.new
+
+    nomination.each do |nom,weight|
```

Comment:
Unused block argument - `weight`. If it's necessary, use `_` or `_weight` as an argument name to indicate that it won't be used.